src/components/dialog-full-screen/dialog-full-screen-test.stories.tsx

```diff
@@ -128,3 +130,37 @@ export const Nested = () => {
 };
 
 Nested.storyName = "nested";
+
+export const WithTwoDifferentNodes = ({
+  children = "This is an example",
+  ...props
+}: Partial<DialogFullScreenProps>) => {
+  const [isOpen, setIsOpen] = useState(true);
+  const ref = useRef<HTMLButtonElement | null>(null);
+  return (
+    <>
+      <DialogFullScreen
+        focusFirstElement={ref}
+        open={isOpen}
+        showCloseIcon
+        title="Example Dialog Full Screen"
+        subtitle={
+          <>
+            <Icon type="add" />
+            <br />
+            Subtitle line 2
+          </>
+        }
+        onCancel={() => setIsOpen(false)}
+        {...props}
+      >
+        <Textbox label="Textbox1" value="Textbox1" />
+        <Textbox label="Textbox2" value="Textbox2" />
+        <Textbox label="Textbox3" value="Textbox3" />
+        <Form>{children}</Form>
+      </DialogFullScreen>
+    </>
+  );
+};
+
+WithTwoDifferentNodes.storyName = "with two different nodes in subtitle";
```

Comment:
**suggestion**: could we enable Chromatic to snapshot this story?

As we are opting to have a visual test for this story instead of writing a Playwright test, this will make sure any visual changes to this story are caught and assessed. 

```suggestion
WithTwoDifferentNodes.storyName = "with two different nodes in subtitle";
WithTwoDifferentNodes.parameters = {
   chromatic: { disableSnapshot: false },
   themeProvider: { chromatic: { theme: "sage" } }
};
```

Reply:
I'm going to approve as once the themes have been removed it's less important to set the chromatic theme imo